packages/block-editor/src/components/block-types-list/index.native.js

```diff
@@ -154,6 +155,7 @@ export default function BlockTypesList( {
 		<SectionList
 			onLayout={ onLayout }
 			testID={ `InserterUI-${ name }` }
+			accessibilityLabel={ __( 'Blocks menu' ) }
```

Comment:
Nitpick: Maybe we could follow the same approach we have on the web using the `label` prop ([reference](https://github.com/WordPress/gutenberg/blob/f93a8f67b8b16eb00e6915555b3f3dac4f54c04d/packages/block-editor/src/components/block-types-list/index.js#L31)).

Reply:
The thing is on the web variant it [renders](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/inserter/block-types-tab.js) the `BlockTypesList` component differently where depending on the case it uses a different label, `Most used`, `Per category`, etc. We show all blocks by default, I think we would need to change the code a bit to be able to have something similar and it might not make sense for this case. What do you think? Currently, the native `BlockTypesList` component is not receiving a `label` value.

Reply:
Yep, it's true that the web version uses different labels because it displays multiple tabs. In our case, we only have two but for consistency, we could consider following a similar approach. My suggestion is to provide the accessibility label via the `label` prop from the tabs where we render the `BlockTypesList` instances:
- https://github.com/WordPress/gutenberg/blob/601789876f7e9749a69b6bf93f86fa201e321a54/packages/block-editor/src/components/inserter/block-types-tab.native.js#L64-L69
- https://github.com/WordPress/gutenberg/blob/601789876f7e9749a69b6bf93f86fa201e321a54/packages/block-editor/src/components/inserter/reusable-blocks-tab.native.js#L29-L34

Similar to what's done on the web for the Reusable tab:
https://github.com/WordPress/gutenberg/blob/601789876f7e9749a69b6bf93f86fa201e321a54/packages/block-editor/src/components/inserter/reusable-blocks-tab.js#L33-L38

Again, not a blocker for the PR, so please feel free to avoid this suggestion.

Reply:
Ahh I see, I did some changes in https://github.com/WordPress/gutenberg/pull/48026/commits/7f99005c6fe3bbf594eeb1638e1cad02b96f20a5 let me know what you think, thanks!

Reply:
Great, thanks for applying the suggestion. The changes look great to me 🎊 !